apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/samples/PairAssertionSamples.kt

```diff
@@ -0,0 +1,53 @@
+package ch.tutteli.atrium.api.fluent.en_GB.samples
+
+import ch.tutteli.atrium.api.fluent.en_GB.first
+import ch.tutteli.atrium.api.fluent.en_GB.second
+import ch.tutteli.atrium.api.fluent.en_GB.toBe
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class PairAssertionSamples {
+
+    // subject under test
+    private val sut = 1 to "one"
```

Comment:
Please duplicate in each test. have a look at the resulting KDoc I guesd it is not included and therefore important context is missing

Reply:
ok